sunpy/net/jsoc/jsoc.py

```diff
@@ -804,14 +742,13 @@ def _lookup_records(self, iargs):
         Do a LookData request to JSOC to workout what results the query returns.
         """
 
-        keywords_default = ['T_REC', 'TELESCOP', 'INSTRUME', 'WAVELNTH', 'CAR_ROT']
         isMeta = iargs.get('meta', False)
         c = drms.Client()
 
         if isMeta:
             keywords = '**ALL**'
         else:
-            keywords = iargs.get('keys', keywords_default)
+            keywords = iargs.get('keys', '**ALL**')
```

Comment:
So if a user adds keys, they will be used here?

Reply:
Yes, now code will work the same way as it worked previously for `a.jsoc.Keys` also specified. I am checking `query_args` in `build_table` for that.